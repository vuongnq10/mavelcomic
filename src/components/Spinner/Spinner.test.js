import React from 'react';
import renderer from 'react-test-renderer';

import Index from './index';

it('Renders Spinner component', () => {
  const index = renderer
    .create(<Index />)
    .toJSON();

  expect(index).toMatchSnapshot();
});