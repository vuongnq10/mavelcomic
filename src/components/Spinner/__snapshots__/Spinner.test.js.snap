// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Spinner component 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
